mbank-cli

```diff
@@ -1213,25 +1213,53 @@ sub do_login  ## no critic (ExcessComplexity)
         $error_code eq ''
             or scraping_error("login.2fa.init.error: $error_code");
         my $auth_mode = $data->{AuthMode} // '';
+        my $content = '';
+        my $tran_id = '';
         if ($auth_mode eq 'NAM') {
-            known_bug(72, '2FA via mobile app is not supported yet');
+            print "Waiting for confirm click in mobile app...\n";
+            $tran_id = $data->{TranId} // '';
+            do {
+                $request = POST(
+                    "$root_url/api/auth/status",
+                    @auth_headers,
+                    'data-type' => 'json',
+                    'Content' => encode_json({
+                        'TranId' => $tran_id
+                    })
+                );
+                $doc = download($request);
+                $data = decode_json($doc->{content})
+                    // scraping_error("login.2fa.exec: $EVAL_ERROR");
+                $error_code = $data->{ErrorCode} // '?';
+                $error_code eq ''
+                    or scraping_error("login.2fa.exec.error: $error_code");
+                if ($data->{Status} eq 'Prepared' || $data->{Status} eq 'PreAuthorized') {
+                    sleep(1);
```

Comment:
PerlCritic complains that the return value from `sleep()` is not checked.
I don't believe that `sleep()` can actually fail on any sane system, but hey, better safe than sorry.